### PR TITLE
Sign up acount type

### PR DESCRIPTION
**Pull request by RomuloLlabresJr**
Hi @ismaeljhon , done adding the new fields for user Table, isFTP, isECommerce,isSocialMedia. 
Thanks

### CODE REVIEW

**Review by ismaeljhon — CHANGES_REQUESTED** [review]
Hi @RomuloLlabresJr kindly retain the env sample.

File: src/tests/categories/team-lead.test.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ describe('Category team lead', () => {
       .expect(res => {
         // check if the user is assigned to the category
         expect(res.body).toHaveProperty('data.createOneCategory.record')
-        expect(res.body.data.createOneCategory.record.teamLead.fullName).toEqual(users[0].fullName)
+        expect(res.body.data.createOneCategory.record.teamLead.firstName).toEqual(users[0].firstName)
         categoryId = res.body.data.createOneCategory.record._id
```
Comment (ismaeljhon) [inline src/tests/categories/team-lead.test.js:58]: @RomuloLlabresJr  Kindly add middleName, lastName on the testing..

File: src/tests/departments/team-lead.test.js [hunk 2]
```diff
@@ -49,7 +49,7 @@ describe('department team lead', () => {
       .expect(res => {
         // check if the user is assigned to the department
         expect(res.body).toHaveProperty('data.createOneDepartment.record')
-        expect(res.body.data.createOneDepartment.record.teamLead.fullName).toEqual(users[0].fullName)
+        expect(res.body.data.createOneDepartment.record.teamLead.firstName).toEqual(users[0].firstName)
```
Comment (ismaeljhon) [inline src/tests/departments/team-lead.test.js:54]: @RomuloLlabresJr  Kindly add middleName, lastName on the testing..

File: src/tests/departments/team-lead.test.js [hunk 3]
```diff
@@ -106,7 +106,7 @@ describe('department team lead', () => {
     })
       .expect(res => {
         expect(res.body).toHaveProperty('data.updateDepartmentById.record')
-        expect(res.body.data.updateDepartmentById.record.teamLead.fullName).toEqual(users[1].fullName)
+        expect(res.body.data.updateDepartmentById.record.teamLead.firstName).toEqual(users[1].firstName)
```
Comment (ismaeljhon) [inline src/tests/departments/team-lead.test.js:115]: @RomuloLlabresJr  Kindly add middleName, lastName on the testing..

File: src/tests/services/project-manager.test.js [hunk 4]
```diff
@@ -50,7 +51,7 @@ describe('service project manager', () => {
     })
       .expect(res => {
         expect(res.body).toHaveProperty('data.createOneService.record.projectManager')
-        expect(res.body.data.createOneService.record.projectManager.fullName).toStrictEqual(defaultProjectManager.fullName)
+        expect(res.body.data.createOneService.record.projectManager.firstName).toStrictEqual(defaultProjectManager.firstName)
```
Comment (ismaeljhon) [inline src/tests/services/project-manager.test.js:55]: @RomuloLlabresJr  Kindly add middleName, lastName on the testing..

File: src/tests/services/project-manager.test.js [hunk 5]
```diff
@@ -107,7 +108,7 @@ describe('service project manager', () => {
     })
       .expect(res => {
         expect(res.body).toHaveProperty('data.createOneService.record.projectManager')
-        expect(res.body.data.createOneService.record.projectManager.fullName).toStrictEqual(newProjectManager.fullName)
+        expect(res.body.data.createOneService.record.projectManager.firstName).toStrictEqual(newProjectManager.firstName)
```
Comment (ismaeljhon) [inline src/tests/services/project-manager.test.js:116]: @RomuloLlabresJr  Kindly add middleName, lastName on the testing..

File: src/tests/services/project-manager.test.js [hunk 6]
```diff
@@ -162,7 +163,7 @@ describe('service project manager', () => {
     })
       .expect(res => {
         expect(res.body).toHaveProperty('data.updateServiceById.record.projectManager')
-        expect(res.body.data.updateServiceById.record.projectManager.fullName).toStrictEqual(updatedProjectManager.fullName)
+        expect(res.body.data.updateServiceById.record.projectManager.firstName).toStrictEqual(updatedProjectManager.firstName)
```
Comment (ismaeljhon) [inline src/tests/services/project-manager.test.js:175]: @RomuloLlabresJr  Kindly add middleName, lastName on the testing..

File: src/tests/services/project-manager.test.js [hunk 7]
```diff
@@ -235,7 +236,7 @@ describe('service project manager', () => {
     })
       .expect(res => {
         expect(res.body).toHaveProperty('data.updateServiceById.record.projectManager')
-        expect(res.body.data.updateServiceById.record.projectManager.fullName).toStrictEqual(defaultProjectManager.fullName)
+        expect(res.body.data.updateServiceById.record.projectManager.firstName).toStrictEqual(defaultProjectManager.firstName)
```
Comment (ismaeljhon) [inline src/tests/services/project-manager.test.js:252]: @RomuloLlabresJr  Kindly add middleName, lastName on the testing..

**Review by ismaeljhon — CHANGES_REQUESTED** [review]
@RomuloLlabresJr Please add these.. thanks

File: src/tests/services/project-manager.test.js (hunk 6 above)
Comment (RomuloLlabresJr) [reply src/tests/services/project-manager.test.js:175]: noted on this. I  will update the code. Thanks

File: src/tests/services/project-manager.test.js (hunk 7 above)
Comment (RomuloLlabresJr) [reply src/tests/services/project-manager.test.js:252]: noted on this. I  will update the code. Thanks

File: src/tests/departments/team-lead.test.js (hunk 2 above)
Comment (RomuloLlabresJr) [reply src/tests/departments/team-lead.test.js:54]: noted on this. I  will update the code. Thanks

File: src/tests/categories/team-lead.test.js (hunk 1 above)
Comment (RomuloLlabresJr) [reply src/tests/categories/team-lead.test.js:58]: noted on this. I  will update the code. Thanks

File: src/schemas/user.js [hunk 8]
```diff
@@ -4,9 +4,20 @@ const faker = require('faker')
...
+    default: ''
+  },
+  middleName: {
+    type: String,
+    require: true,
+    default: ''
+  },
+  lastName: {
+    type: String,
+    require: true,
```
Comment (ismaeljhon) [inline src/schemas/user.js]: @RomuloLlabresJr it seems this is "required".. please also the other fields

**Review by ismaeljhon — CHANGES_REQUESTED** [review]

File: src/schemas/user.js (hunk 8 above)
Comment (RomuloLlabresJr) [reply src/schemas/user.js]: done updating the code. it is fixed now. thanks

**Pull request merged**